skylark/skylark.bzl

```diff
@@ -29,15 +33,9 @@ def _get_transitive_sources(deps):
   """
```

Comment:
This `_get_transitive_sources()` function is now used in only one place.  You should probably just inline it.

Also, rather than repeatedly adding things to a depset(), which creates many unnecessary depset() objects, you should just do:

```python
transitive_srcs = depset(transitive=[
    dep[SkylarkLibraryInfo].transitive_srcs for dep in deps
])
```